AbstractSQLRules2SQL.ometajs

```diff
@@ -743,6 +744,23 @@ export ometa AbstractSQLRules2SQL {
 			)
 		),
 
+	Case :indent =
+		[
+			'Case'
+			[	When(indent)+:whens
+			]
+			AnyValue(indent):val
+		]
+		-> (' CASE ' + whens.join(' ') + ' ELSE ' + val + ' END'),
```

Comment:
We have the `indent`/`NestedIndent` stuff which might be nicer to use in order to print the sql as
```sql
SELECT CASE
		WHEN a THEN n
		WHEN b THEN o
		WHEN c THEN p
		WHEN d THEN q
		WHEN e THEN r
		WHEN f THEN s
		ELSE z
	END
FROM ...
```
instead of
```sql
SELECT CASE WHEN a THEN n WHEN b THEN o WHEN c THEN p WHEN d THEN q WHEN e THEN r WHEN f THEN s ELSE z END
FROM ...
```
in order to make debugging a little easier on us humans

Reply:
Good idea. I will try and look to other examples in the OMetaJS to see how to do this, but I am presuming that `nestedIndent` is just a variable for `\t` etc?

Reply:
`NestedIndent(indent):nestedIndent` just adds an extra level of nesting (`\t`), and so will be something like `\n\t\t`.  So you should be able to do `.join(nestedIndent)` `+ nestedIndent +` etc in the various places to format it nicely - and passing the `nestedIndent` into the child functions to propogate the nesting level.
Overall the indentation is just a nice to have, but it is super nice for debugging